commit f6d33b3a2d1077c91425819de145ba2e5ae3cd87
Author: Raphael Kubo da Costa <rakuco@FreeBSD.org>
Date:   Fri Aug 15 13:21:21 2014 +0300

    FindFFmpeg: Use the right name for the pkg-config component variables.
    
    Commit 3194a35e ("Rewritten FindFFmpeg.cmake") introduced a regression
    in the find_component macro: the pkg_check_module() call uses
    PC_${component} as the variable name, while the calls to find_path() and
    find_library() use PC_LIB${component} instead.
    
    Fix the calls to find_path() and find_library().
    
    Based on a patch by Thomas Klausner <tk@giga.or.at>, thanks a lot!
    
    BUG:      338266
    CCMAIL:   tk@giga.or.at
    FIXED-IN: 4.14.1

diff --git a/cmake/modules/FindFFmpeg.cmake b/cmake/modules/FindFFmpeg.cmake
index 4e1d50e..27c6b16 100644
--- a/cmake/modules/FindFFmpeg.cmake
+++ b/cmake/modules/FindFFmpeg.cmake
@@ -68,16 +68,16 @@ macro(find_component _component _pkgconfig _library _header)
 
   find_path(${_component}_INCLUDE_DIRS ${_header}
     HINTS
-      ${PC_LIB${_component}_INCLUDEDIR}
-      ${PC_LIB${_component}_INCLUDE_DIRS}
+      ${PC_${_component}_INCLUDEDIR}
+      ${PC_${_component}_INCLUDE_DIRS}
     PATH_SUFFIXES
       ffmpeg
   )
 
   find_library(${_component}_LIBRARIES NAMES ${_library}
       HINTS
-      ${PC_LIB${_component}_LIBDIR}
-      ${PC_LIB${_component}_LIBRARY_DIRS}
+      ${PC_${_component}_LIBDIR}
+      ${PC_${_component}_LIBRARY_DIRS}
   )
 
   set(${_component}_DEFINITIONS  ${PC_${_component}_CFLAGS_OTHER} CACHE STRING "The ${_component} CFLAGS.")
